### DISCUSSION TITLE
honeypot attacks using Backdoor.Windows.CMD.Shell

### DISCUSSION THREAD

**Discussion by DubovskoySergey** [telekom-security/tpotce#851] [Q&A] [answered]
Hi! Recently, we noticed that our honeypot was performing attacks using Backdoor.Windows.CMD.Shell. The firewall logs indicate that attacks are happening through port 4444 or 21000. Is this some new functionality or has our honeypot been hacked? In the system itself, there is no information about the processes using these ports. Tell me what where to look for the problem?
We are using the latest stable release of Debian.
Also the latest version of your TPotce
![Screenshot from 2021-06-15 08-43-25](https://user-images.githubusercontent.com/34303263/121999414-f4840700-cdb5-11eb-81db-69188a966594.png)

**Comment by t3chn0m4g3** [comment, marked as answer]
Based on the screenshot I cannot really assess what I am looking at. However Cowrie allows outgoing connections to some extent in order to store malware files and then terminates the connection. These connections are in fact logged as part of the Cowrie honeypot and you can visualize them in Kibana.

**Reply by DubovskoySergey** [reply]
Thanks for the answer. This is what I wanted to find out.
